doc/dune-files.rst

```diff
@@ -55,6 +57,18 @@ Sets the version of the project:
 
     (version <version>)
 
+cram
```

Comment:
Since it's a section header, I've proposed we capitalize these, but I haven't yet heard if those will be accepted or not.

Reply:
I'll leave this as is for now then, and you and the team can change them all if you end up decided that. I don't think we'd just want this one header capitalized :)

fwiw, I'd find it confusing as a reader if these headers were capitalized, since they correlate to the keywords used in stanzas, and those are not capitalized. But perhaps they should use monospaced text (if that's allowed in headers in rst?).

Reply:
Understood! I got the same feedback from another engineer, so it’s fine to keep them all lowercase.